security-council-rehearsals/README.md

```diff
@@ -5,6 +5,7 @@ This directory contains templates and ceremonies related to onboarding Security
 ## Creating and signing a new rehearsal ceremony
 
 To create a new rehearsal ceremony, follow the instructions in the _Facilitator_ section of the README files for each of the following rehearsals:
+
```

Comment:
nit, can undo this diff since we don't touch the rehearsals dir anywhere else here